tools/harness-simulation/README.md

```diff
@@ -0,0 +1,28 @@
+# Test Harness on Simulation Environment Setup
+
+THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template `IThci`, which is used by the Thread Test Harness Software to control OpenThread-based reference devices according to each test scenario.
+
+SI (Sniffer Interface) is an implementation of the sniffer abstract class template `ISniffer`, which is used by the Thread Test Harness Software to sniff all packets sent by devices.
+
+Both OpenThread simulation and sniffer simulation are required to run on a POSIX environment. However, Harness has to be run on Windows, which is a non-POSIX environment. So both two systems are needed, and their setup procedures in detail are listed in the following chapters. Either two machines or one machine running two (sub)systems (e.g., VM, WSL) is feasible.
+
+Platform developers should modify the THCI implementation and/or the SI implementation directly to match their platform (for example, the path of the OpenThread repository).
+
+## POSIX Environment Setup
+
+1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. For example, run `script/cmake-build simulation` in the top directory of OpenThread. Then `ot-cli-ftd` is built in the directory `build/simulation/examples/apps/cli/`.
+
+## Test Harness Environment Setup
+
+1. Run `harness\install.bat` on the machine which installed Harness.
+2. Check the configuration file `C:\GRL\Thread1.2\Thread_Harness\simulation\config.py`
+   - Edit the value of `REMOTE_USERNAME` to the username expected to connect to on the remote POSIX environment.
+   - Edit the value of `REMOTE_PASSWORD` to the password corresponding to the username above.
+   - Edit the value of `REMOTE_OT_PATH` to the absolute path where the top directory of the OpenThread repository is located.
+3. Add the additional simulation device information in `harness\Web\data\deviceInputFields.xml` to `C:\GRL\Thread1.2\Web\data\deviceInputFields.xml`.
+
+## Run Test Harness on Simulation
+
+1. On POSIX machine, switch to the directory `posix`, and run `python harness_dev_discovery.py --interface=INTERFACE --ot1.1=OT11_NUM --sniffer=SNIFFER_NUM` (e.g. `python harness_dev_discovery.py --interface=eth0 --ot1.1=24 --sniffer=2` starts 24 OT FTD simulations and 2 sniffer simulations and can be discovered on eth0).
```

Comment:
At minimum, move the commands into a code block so it's more clear what the user is expected to run.  See earlier comment for an example.